extensions/sdk/package.json

```diff
@@ -34,23 +34,22 @@
   "devDependencies": {
     "@types/jest": "^26.0.24",
     "@types/uuid": "^8.3.1",
-    "@types/webextension-polyfill": "^0.8.0",
+    "@types/webextension-polyfill": "^0.8.3",
     "@typescript-eslint/eslint-plugin": "^5.17.0",
     "@typescript-eslint/parser": "^5.17.0",
     "eslint": "^8.12.0",
     "jest": "^27.0.6",
     "jest-transform-stub": "^2.0.0",
-    "jest-webextension-mock": "^3.7.17",
+    "jest-webextension-mock": "github:rhelmer/jest-webextension-mock#rally",
```

Comment:
@thomik-corp do you have thoughts on how we should vendor this? I have two patches to do what we need:

- make `tabs.reload()` work in promise-mode
- implement the basic `windows.*` API

Should I create a repo in the mozilla-rally org, or have this package live in the `rally` monorepo? wdyt?

Eventually I'd like to get this upstream and we'll switch back to the published npm package.

Reply:
We have several options here. We can either put it as a private repo in monorepo, never publish it and then swap it later (we can dump it in external/lib/ folder or something) or create a new repo under Mozilla. I'd say we should go with former as we don't want others to consume off mozilla?